include/mlir-tcp/Dialect/IR/TcpOps.td

```diff
@@ -389,6 +389,44 @@ def Tcp_FloorOp : Tcp_UnaryElementwiseOp<"floor", [SameOperandsAndResultElementT
   let assemblyFormat = "$in attr-dict `:` type($in) `->` type($out)";
 }
 
+def Tcp_RoundOp : Tcp_UnaryElementwiseOp<"round", [SameOperandsAndResultElementType]> {
+  let summary = "Computes elementwise rounding of input, with ties rounded away from zero";
+
+  let description = [{
+    Computes elementwise rounding of the input float tensor to the nearest integer value
+    in float, with halfway values rounded away from zero.
+  }];
+
+  let arguments = (ins
+    Tcp_FloatTensor:$in
+  );
+
+  let results = (outs
+    Tcp_FloatTensor:$out
+  );
+
+  let assemblyFormat = "$in attr-dict `:` type($in) `->` type($out)";
+}
+
+def Tcp_RoundEvenOp : Tcp_UnaryElementwiseOp<"round_even", [SameOperandsAndResultElementType]> {
+  let summary = "Computes elementwise rounding of input, with ties rounded to even";
```

Comment:
nit: `... with ties rounded to nearest even`

Reply:
Done